SPECS/xen.spec

```diff
@@ -19,9 +19,9 @@ Version: 4.13.4
 # the xen_extra field can't hold more than 16 chars
 # so instead of using %%release to define XEN_VENDORVERSION
 # we create a base_release macro, that doesn't contain the dist suffix
-%define base_release 9.24.2
+%define base_release 9.25.1
 Release: %{base_release}%{?dist}
-License: GPLv2 and LGPLv2+ and BSD
+License: GPLv2 and LGPLv2+ and MIT and Public Domain
```

Comment:
This still differs from upstream's License tag. Don't forget to compare with the XS-8.2 branch and make sure all the changes are relevant.